doc/develop/test/twister.rst

```diff
@@ -1124,6 +1124,43 @@ using an external J-Link probe.  The ``probe_id`` keyword overrides the
       runner: jlink
       serial: null
 
+Additional Scripts
+++++++++++++++++++
+
+Twister offers users the flexibility to automate the execution of external
+scripts at precise moments. These scripts can be strategically deployed in
+three distinct phases: pre-script, post-flash-script and post-script.
+This functionality could help configuring the environment optimally
+before testing.
+
+To leverage the scripting capability, users must append the argument
```

Comment:
This sentence is confusing not mentioning that pre- and post- scripts are also (and already) configurable for a device as hardware map parameters.

In general, there is no explanation of how the proposed feature extends device attributes and in what context: is it for `--device-testing`` only ?